cxx-lint/src/main/java/org/sonar/cxx/cxxlint/CppLanguage.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Sonar C++ Plugin (Community)
+ * Copyright (C) 2010-2017 SonarOpenCommunity
+ * http://github.com/SonarOpenCommunity/sonar-cxx
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.cxx.cxxlint;
+
+import org.sonar.api.config.Settings;
+import org.sonar.cxx.CxxLanguage;
+
+/**
+ *
+ * @author jocs
+ */
+public class CppLanguage extends CxxLanguage {
+
+  public CppLanguage() {
+    super("c++", new Settings());
+  }  
+  
+  @Override
+  public String[] getFileSuffixes() {
+    return new String [] {"cpp", "hpp", "h", "hxx", "cxx"};

```

Comment:
yep, but it takes less code to do this. we on the next pr we should clarify a bit better this behaviour